client/package.json

```diff
@@ -0,0 +1,112 @@
+{
+  "name": "mailhold-ui-vue",
+  "version": "1.0.0",
+  "description": "Vue.js UI for Mailhold",
```

Comment:
holdmail, jees. :)

Reply:
I don't know why I always do this, it stuck in the beginning and it's a hard habit to shake.